### PR TITLE
TM-6757 Convert the nginx logs to json format

### PR DESCRIPTION
**Pull request by sydboraa**
1- If it is a real user: 
Screenshot 2020-01-22 at 14 02 42

2- And for the AWS requests:
Screenshot 2020-01-22 at 14 04 14

Screenshot 2020-01-22 at 14 05 07

@anho

### CODE REVIEW

File: nginx-proxy-node/1.17.1/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    '"request_method": "$request_method", '
+    '"http_referrer": "$http_referer", '
+    '"http_user_agent": "$http_user_agent" }';
+
+  access_log  /var/log/nginx/access.log  json;
+
+  server {
+    listen 3000;
+
+    # NGINX will reject anything not matching /api
```
Comment (franckbrignoli) [inline nginx-proxy-node/1.17.1/nginx.conf]: is it really the case? ;)

File: nginx-proxy-node/1.17.1/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  # NGINX will handle gzip compression of responses from the app server
+  gzip on;
+  gzip_proxied any;
+  gzip_types application/javascript application/json text/css;
+  gzip_min_length 1000;
+  server_tokens off;
+
+  log_format json '{ "timestamp": "$time_iso8601", '
+    '"host": "$host", '
+    '"remote_addr": "$remote_addr", '
```
Comment (franckbrignoli) [inline nginx-proxy-node/1.17.1/nginx.conf]: will that be the correct IP address and not the one from the ALB ? or varnish?

File: nginx-proxy-node/1.17.1/nginx.conf (hunk 1 above)
Comment (sydboraa) [reply nginx-proxy-node/1.17.1/nginx.conf]: I took that code from here https://github.com/trademachines/docker-images/blob/master/nginx-proxy-node/1.17/nginx.conf

File: nginx-proxy-node/1.17.1/nginx.conf (hunk 2 above)
Comment (sydboraa) [reply nginx-proxy-node/1.17.1/nginx.conf]: Good catch. it is from ALB. I will try to get real ip.

File: nginx-proxy-node/1.17.1/nginx.conf (hunk 1 above)
Comment (franckbrignoli) [reply nginx-proxy-node/1.17.1/nginx.conf]: it looks like a bad copy/pasting from @anho though

File: nginx-proxy-node/1.17.1/nginx.conf (hunk 2 above)
Comment (sydboraa) [reply nginx-proxy-node/1.17.1/nginx.conf]: done

**Pull request merged**